content/md/en/docs/tutorials/build-a-parachain/connect-a-local-parachain.md

```diff
@@ -44,10 +44,10 @@ To build the parachain template:
 
 2. Clone the branch of the `substrate-parachain-template` repository that matches the release branch you used to configure the relay chain.
    
-   For example, if you used the `release-v0.9.37` Polkadot release branch to configure the local relay chain, use the `polkadot-v0.9.37` branch for the parachain template.
+   For example, if you used the `release-v1.0.0-cumulus`Polkadot release branch to configure the local relay chain, use the `parachains-v10000-rc5` branch for the parachain template.
```

Comment:
We could leave this example pointing to `v1.0.0` instead of a release candidate or other more concrete branches. I would say it gives a good enough idea of what users should be looking for, and how to extrapolate that if willing to work with different versions to the one showcased in the guide.

Reply:
yeah I did this assuming that backports pending would break things, but AFAIK 1.0.0 has all those now and have moved to these. Thanks for testing!!